Tests/TestingApps/PurchaseTesterSwiftUI/Shared/Views/ConfigurationView.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  ConfigurationView.swift
+//  PurchaseTester
+//
+//  Created by Nacho Soto on 10/25/22.
+//
+
+import SwiftUI
+
+import RevenueCat
+
+struct ConfigurationView: View {
+
+    struct Data: Equatable, Codable {
+        var apiKey: String = Constants.apiKey.replacingOccurrences(of: "REVENUECAT_API_KEY",
+                                                                   with: "")
+        var proxy: String = ""
+        var storeKit2Enabled: Bool = true
+    }
+
+    let onContinue: (Data) -> Void
+
+    init(onContinue: @escaping (Data) -> Void) {
+        self.onContinue = onContinue
+
+        if let data = self.storedData,
+           let decoded = try? JSONDecoder().decode(Data.self, from: data) {
+            self._data = .init(initialValue: decoded)
+        }
+    }
+
+    var body: some View {
+        Form {
+            Section(header: Text("Configuration")) {
+                TextField("API Key (required)", text: self.$data.apiKey)
+
+                TextField("Proxy URL (optional)", text: self.$data.proxy)
+            }
+
+            Section {
+                Toggle(isOn: self.$data.storeKit2Enabled) {
+                    Text("StoreKit2 enabled")
+                }
+            }
+
+            Section(header: Text("About")) {
+                HStack {
+                    Text("Version")
+                    Spacer()
+                    Text(Purchases.frameworkVersion)
+                }
+            }
+        }
+        .textInputAutocapitalization(.never)
+        .autocorrectionDisabled(true)
+        .navigationTitle("Purchase Tester")
+        .toolbar {
+            ToolbarItem(placement: .navigationBarTrailing) {
+                Button {
+                    self.saveData()
+                    self.onContinue(self.data)
+                } label: {
+                    Text("Continue")
+                }
+                .disabled(!self.contentIsValid)
+            }
+        }
+        .onChange(of: self.data) { _ in
+            self.saveData()
```

Comment:
A UX question, should we only save data when pressing the "Continue" button? I think it shouldn't matter much, but seems more intuitive for me... For example, if a user wants to reconfigure the sdk, starts changing the data but then restarts the app, I would expect to still have the old information... NABD though.

Reply:
Yeah, I could see either way. I don't think this is that "wrong", so if you don't mind I'm going to leave it as is so I don't have to re-test this 9-day old PR.